### ISSUE TITLE
Error on http://boowakwala.uptoten.com/kids/boowakwala-club-flower-windmilldress.html 

### ISSUE THREAD

**Issue by BoowaKwala** [ruffle-rs/ruffle#7899]
# Error Info
Error name: Error
Error message: panicked at 'No script object for display object', core/src/avm1.rs:230:18
Error stack:
```
Error: panicked at 'No script object for display object', core/src/avm1.rs:230:18
    at e.wbg.__wbg_new_df6e6ab7a65c4c4d (http://www.uptoten.com/js/ruffle/core.ruffle.dc6a4efed31836d31c04.js:1:5745)
    at std::sync::once::Once::call_once::{{closure}}::h2b0dd13dece8b2bd (wasm://wasm/016ae5c6:wasm-function[934]:0x26ac3f)
    at std::sync::once::Once::call_inner::h00971925aa1adad3 (wasm://wasm/016ae5c6:wasm-function[1370]:0x2f96e0)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hcedf29f658f02358 (wasm://wasm/016ae5c6:wasm-function[4798]:0x441c82)
    at std::panicking::rust_panic_with_hook::h84feca33bd4bd229 (wasm://wasm/016ae5c6:wasm-function[3519]:0x408f9d)
    at core::panicking::panic_fmt::hc171d095bc4a492d (wasm://wasm/016ae5c6:wasm-function[4975]:0x447541)
    at <ruffle_core::display_object::DisplayObject as ruffle_core::display_object::TDisplayObject>::post_instantiation::he236c9569f8d46d0 (wasm://wasm/016ae5c6:wasm-function[1883]:0x360e27)
    at ruffle_core::display_object::movie_clip::MovieClip::instantiate_child::h71d9de9bd3da2cf2 (wasm://wasm/016ae5c6:wasm-function[456]:0x161d0e)
    at ruffle_core::display_object::movie_clip::MovieClip::place_object::h59b8c26eb348ef31 (wasm://wasm/016ae5c6:wasm-function[925]:0x268745)
    at <ruffle_core::display_object::movie_clip::MovieClip as ruffle_core::display_object::TDisplayObject>::construct_frame::h289fc6ea77915fdf (wasm://wasm/016ae5c6:wasm-function[393]:0x11a2c3)
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: http://boowakwalastatic.uptoten.com/bk_game_loader.swf?v=1
Param movie: http://boowakwalastatic.uptoten.com/bk_game_loader.swf?v=1
Param quality: high
Param bgcolor: #FFFFCC
Param play: true
Param loop: true
Param scale: exactfit
Param menu: false
Param devicefont: false
Param allowScriptAccess: always
Param wmode: opaque
Param base: http://boowakwalastatic.uptoten.com/
Param FlashVars: game=swf/en/dressingup/windmillDress.swf
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined
Attribute 5: undefined

# Page Info
Page URL: http://boowakwala.uptoten.com/kids/boowakwala-club-flower-windmilldress.html
SWF URL: http://boowakwalastatic.uptoten.com/bk_game_loader.swf?v=1

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-08-30
Channel: nightly
Built: 2022-08-30T00:18:51.591Z
Commit: 36450619109b6ed510991b311f4888c0784eba8e
Is extension: false

# Metadata
width: 800
height: 500
frameRate: 12
numFrames: 1
swfVersion: 15
backgroundColor: #FFFFFF
isActionScript3: true
uncompressedLength: 6041

**Comment by n0samu** [comment]
This error occurs because the page has an AVM2 loader SWF that loads an AVM1 SWF. Ruffle does not support running AVM1 SWFs through AVM2 loaders yet. But loading the main AVM1 SWF directly in Ruffle works fine: http://boowakwala.uptoten.com/swf/en/dressingup/windmillDress.swf
If you are the owner of this website, you may want to recreate your loader SWF in ActionScript 2. I think that would be enough to get most of the site working.

**Comment by n0samu** [comment]
Fixed by #11005. If you are the site owner, you should update your version of Ruffle after the next nightly (which will be available shortly after midnight UTC).
